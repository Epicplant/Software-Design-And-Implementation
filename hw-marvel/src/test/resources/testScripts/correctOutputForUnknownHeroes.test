# Tests when the start and/or the dest node don't exist that the correct output is displayed

# Loads a new graph from a csv file
LoadGraph heroes complexHeroGraph.csv

# Finds shortest path between heroes
FindPath heroes A Z

# Finds shortest path between heroes
FindPath heroes Z A

# Finds shortest path between heroes
FindPath heroes X Z