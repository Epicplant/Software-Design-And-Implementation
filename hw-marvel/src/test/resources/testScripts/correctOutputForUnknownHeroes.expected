# Tests when the start and/or the dest node don't exist that the correct output is displayed

# Loads a new graph from a csv file
loaded graph heroes

# Finds shortest path between heroes
unknown: Z

# Finds shortest path between heroes
unknown: Z

# Finds shortest path between heroes
unknown: X
unknown: Z